content/blog/2022-07-26-countr-upgrade-maintenance-announcement.mdx

```diff
@@ -0,0 +1,49 @@
+---
+authors: promise
+tags: [maintenance, improvements]
+---
+
+# Countr Upgrade + Maintenance Announcement
+
+We are doing a massive upgrade on Countr that we've planned for way over a year. It's finally here. Global maintenance will take place on August 5th until August 7th, and will affect all services including Countr, Countr Premium, Website and Status page. We will most likely not use all three days for everything, and we hope to have the bot up on the 5th. Other services might take more time.
+
+<!--truncate-->
+
+## Visible changes
+
+* **Slash commands:** We're finally moving over to slash commands! There will be some @mention commands, but these are mostly for the support team of the bot to help you if you need help.
```

Comment:
Maybe replace one of the help's with a form of assist to avoid repetitiveness eg -  of the bot to assist you if you need help

Reply:
wdym?